### PR TITLE
feat: add plugin All Blue

### PR DESCRIPTION
**Pull request by z384647062**
closes #

### CODE REVIEW

**Comment by wenluomask** [comment]
1. For the wrong network, we do have a button component you can use to switch network;
Screen Shot 2021-11-29 at 1 49 40 PM
Screen Shot 2021-11-29 at 1 53 43 PM

2. The plugin card will not refresh status if I change network or wallet, it should be monitor in time;(Maybe the first issue can help)
Screen Shot 2021-11-29 at 1 57 39 PM

3. After connect the wallet, the plugin card will not display the context, I need to refresh;
Screen Shot 2021-11-29 at 3 39 42 PM

4. For the incorrect url(like https://findtruman.io/#/encryption?payload=eyJjaWQiOiAiMDAwMDAxIiwgICJkYXRhIjogIuWXs), the plugin card should not be rendered;
Screen Shot 2021-11-29 at 5 10 34 PM

5. If user cancel the sign from Mask wallet, it will stay in submitting status;(But it works in Metamask)
Screen Shot 2021-11-29 at 6 04 30 PM

6. Hard to test the result of story or poll, please test by yourself or provide me more situations;
Screen Shot 2021-11-29 at 6 23 12 PM


7. UI bugs:
- If option is selected, the color of text should change to white in light mode;
Screen Shot 2021-11-29 at 4 00 34 PM

- There should be a background color for buy button and some typo errors;
Screen Shot 2021-11-29 at 4 51 32 PM

-For the poll didn't selected, use the tag "Select" and then change to "Selected" after selection;
Screen Shot 2021-11-29 at 5 53 50 PM

**Comment by wenluomask** [comment]
- [ ] 8. Failed to submit poll by using Mask Wallet;
Screen Shot 2021-11-30 at 2 40 09 PM

- [ ] 9. The old poll is showing incorrect data;
You can check your tweet: https://twitter.com/ZhouMask/status/1465160080022667265
Screen Shot 2021-11-30 at 2 42 36 PM

- [ ] 10. Change network may cause the option's  selected UI disappear;
Screen Shot 2021-11-30 at 2 26 32 PM

**Comment by wenluomask** [comment]
Checked.

File: packages/mask/src/plugins/Allblue/SNSAdaptor/OptionsCard.tsx [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+    //                 )
+    //                 href = newHref
+    //             }
+    //         }, 500)
+    //     }, 1000)
+    //     return () => {
+    //         interval && clearInterval(interval)
+    //     }
+    // }, [])
+
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Allblue/SNSAdaptor/OptionsCard.tsx:130]:
```suggestion
```

File: packages/mask/src/plugins/Allblue/SNSAdaptor/OptionsCard.tsx [hunk 2]
```diff
@@ -0,0 +1,441 @@
...
+}
+export default function OptionsCard(props: OptionsViewProps) {
+    const { type, userStatus, onSubmit } = props
+    const [selected, setSelected] = useState<boolean>(true)
+    const [choice, setChoice] = useState<number>(userStatus ? userStatus.choice : -1)
+    const [submitting, setSubmitting] = useState<boolean>(false)
+    const [error, setError] = useState<'' | 'unsupported-chain' | 'insufficient-nft'>('')
+    const [unmeetCondition, setUnmeetCondition] = useState<PuzzleCondition[]>([])
+    const [snackVisible, setSnackVisible] = useState<boolean>(false)
+    // const [verticalStepper] = useState<boolean>(document.body.clientWidth < 420)
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Allblue/SNSAdaptor/OptionsCard.tsx:94]:
```suggestion
```

File: packages/mask/src/plugins/Allblue/SNSAdaptor/OptionsCard.tsx [hunk 3]
```diff
@@ -0,0 +1,441 @@
...
+                anchorOrigin={{ vertical: 'top', horizontal: 'center' }}
+                open={snackVisible}
+                onClose={() => setSnackVisible(false)}>
+                <Alert onClose={() => setSnackVisible(false)} variant="filled" severity="error" sx={{ width: '100%' }}>
+                    {t('plugin_allblue_submit_failed')}
+                </Alert>
+            </Snackbar>
+            {userStatus && (
+                <>
+                    {/*{type === AllbluePostType.Poll && renderStepper(userStatus, verticalStepper)}*/}
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Allblue/SNSAdaptor/OptionsCard.tsx:412]:
```suggestion
```

File: packages/mask/src/plugins/Allblue/SNSAdaptor/ResultCard.tsx [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+export const BorderLinearProgress: any = styled(LinearProgress)(({ theme }) => ({
+    height: 10,
+    borderRadius: 5,
+    [`&.${linearProgressClasses.colorPrimary}`]: {
+        backgroundColor: theme.palette.grey[theme.palette.mode === 'light' ? 200 : 800],
+    },
+    [`& .${linearProgressClasses.bar}`]: {
+        borderRadius: 5,
+        // backgroundColor: theme.palette.mode === 'light' ? '#1a90ff' : '#308fe8',
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Allblue/SNSAdaptor/ResultCard.tsx:27]:
```suggestion
```

File: packages/mask/src/plugins/Allblue/Worker/apis/index.ts [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+        method: 'GET',
+    })
+}
+
+// export function fetchUserStageStatus(stageId: string, address: string): Promise<UserStageStatus> {
+//     return request<UserStageStatus>(`${ALLBLUE_API_PREFIX}/stages/${stageId}/status?uaddr=${address}`, {
+//         method: 'GET',
+//     })
+// }
+
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Allblue/Worker/apis/index.ts:63]:
```suggestion
```

**Comment by guanbinrui** [comment]
Hi @z384647062, please remove those commented lines and fix the Check Spell (Learn more by searching `cspell.json` in the codebase).

**Comment by guanbinrui** [comment]
Replaced by #5031.

**Pull request closed without merging**